src/cms/src/api/park-contact/content-types/park-contact/lifecycles.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+const updateName = async (data, where) => {
```

Comment:
The name only gets updated when you update a contact, not when you add a new contact.  Is that consistent with how other collections work? I thought it was broken at first

Reply:
It works when you add a new park-contact. `name` will be automatically created when the publish button is clicked. It's the same function as park-activity and park-facility. Please let me know if it doesn't work on your end.

Reply:
Sounds good. I didn't press publish